### PR TITLE
add script and corresponding readme

### PR DESCRIPTION
**Pull request by amdomanska**
#Closes: https://github.com/DOAJ/doajPM/issues/2056#issuecomment-544894778

### CODE REVIEW

**Comment by amdomanska** [comment]
@Steven-Eardley could you please let me know when I could run this migration on live?

**Comment by amdomanska** [comment]
Improved. Thank you guys for your help, I really overcomplicated that. I removed articles from the script, added suggestions and used wrote simple json and operations to use those with upgrade.py

File: portality/migrate/20191128_2056_keywords_to_lower/operations.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from portality import models
+from portality.core import app
+import esprit
+import re
```
Comment (Steven-Eardley) [inline portality/migrate/20191128_2056_keywords_to_lower/operations.py]: I don't think any of these imports are used

**Comment by amdomanska** [comment]
Now it's gonna be fine I think.

**Comment by amdomanska** [comment]
@Steven-Eardley I guess it's finally ready to run

**Pull request closed without merging**